src/robotide/editor/macroeditors.py

```diff
@@ -51,6 +51,9 @@ def close(self):
     def save(self):
         self.kweditor.save()
 
+    def movecursordown(self):
```

Comment:
This is redundant, of move_cursor_down (and see comment there).

Reply:
OK